client/app/license/migrate/license-migrate.controller.js

```diff
@@ -1,168 +1,160 @@
-angular.module("Module.license.controllers").controller("LicenseMigrateCtrl", [
-    "$scope",
-    "License",
-    "Alerter",
+angular.module("Module.license").controller("LicenseMigrateCtrl", ($scope, $translate, License, Alerter) => {
+    "use strict";
 
-    function ($scope, License, Alerter) {
-        "use strict";
+    $scope.availableIpBlock = {};
 
-        $scope.availableIpBlock = {};
+    $scope.loaders = {
+        ips: false,
+        canBeMovedTo: false,
+        migrate: false
+    };
 
-        $scope.loaders = {
-            ips: false,
-            canBeMovedTo: false,
-            migrate: false
-        };
+    $scope.selected = {
+        ipBlock: null,
+        ip: null
+    };
 
-        $scope.selected = {
-            ipBlock: null,
-            ip: null
-        };
+    $scope.ipValid = {
+        value: false
+    };
 
-        $scope.ipValid = {
-            value: false
-        };
+    $scope.ipCanBeMoved = {
+        value: null
+    };
 
-        $scope.ipCanBeMoved = {
-            value: null
-        };
+    if ($scope.currentActionData && $scope.currentActionData.license) {
+        $scope.license = $scope.currentActionData.license;
+    }
 
-        if ($scope.currentActionData && $scope.currentActionData.license) {
-            $scope.license = $scope.currentActionData.license;
-        }
+    function checkIpValidity () {
+        $scope.ipCanBeMoved.value = null;
+        $scope.loaders.canBeMovedTo = true;
+        return License.canLicenceBeMovedTo({
+            type: $scope.license.type,
```

Comment:
You remove the `angular.lowercase()` on the `License` service. Are you sure about that change?